build/Build.cs

```diff
@@ -93,7 +103,7 @@ class Build : NukeBuild
         .Executes(() =>
         {
             DotNetTest(_ => _
-                .SetProjectFile(Solution)
```

Comment:
Using solution did not work neither did calling the halibut test project directly :(